### ISSUE TITLE
Connecting to www.dropbox.com (www.dropbox.com)|128.121.243.106|:443... failed: Connection timed out

### ISSUE THREAD

**Issue by Xuhd08** [mlcommons/ck#1371]
when i build docker use Dcokerfile, the CK error unable to resolve

command:
RUN . /opt/vitis_ai/conda/etc/profile.d/conda.sh &&     conda activate vitis-ai-tensorflow &&     cd /opt/tvm-vai// &&    python -m ck pull repo:ck-env &&     python -m ck install package:imagenet-2012-val-min

failed log:

Installing to /opt/tvm-vai/CK-TOOLS/dataset-imagenet-ilsvrc2012-val-min
72.81 
72.81 
72.81 Checking whether '/opt/tvm-vai/CK-TOOLS/dataset-imagenet-ilsvrc2012-val-min/ILSVRC2012_img_val.tar' already exists ...
72.81 
72.81 Downloading ILSVRC'12 validation dataset from 'https://www.dropbox.com/s/57s11df6pts3z69/ILSVRC2012_img_val_500.tar' ...
72.81 --2024-12-01 18:06:29--  https://www.dropbox.com/s/57s11df6pts3z69/ILSVRC2012_img_val_500.tar
72.81 Resolving www.dropbox.com (www.dropbox.com)... 128.121.243.106, 2a03:2880:f10c:83:face:b00c:0:25de
72.82 Connecting to www.dropbox.com (www.dropbox.com)|128.121.243.106|:443... failed: Connection timed out.
202.5 Connecting to www.dropbox.com (www.dropbox.com)|2a03:2880:f10c:83:face:b00c:0:25de|:443... failed: Network is unreachable.
202.5 Error: Downloading ILSVRC'12 validation set from 'https://www.dropbox.com/s/57s11df6pts3z69/ILSVRC2012_img_val_500.tar' failed!
202.6 
202.6    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
202.6    CK detected a PROBLEM in the third-party CK package:
202.6 
202.6    CK package:           imagenet-2012-val-min
202.6    CK repo:              ck-env
202.6    CK repo URL:          https://github.com/ctuning/ck-env
202.6    CK package URL:       https://github.com/ctuning/ck-env/tree/master/package/imagenet-2012-val-min
202.6    Issues URL:           https://github.com/ctuning/ck-env/issues
202.6 
202.6    CK stable package URL: https://cKnowledge.io/c/package/imagenet-2012-val-min
202.6 
202.6    Please, check that there is no discussion about this issue at https://cKnowledge.io/c/package/imagenet-2012-val-min
202.6 
202.6    Please, submit the original comand line and the *full* log to:
202.6      * https://github.com/mlcommons/ck/issues
202.6      * https://github.com/mlcommons/ck-mlops/issues
202.6    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
202.6 
202.6 CK error: [package] package installation failed!
------
Dockerfile.vitis_ai:100
--------------------
  99 |     ENV HOME $TVM_VAI_HOME
 100 | >>> RUN . $VAI_ROOT/conda/etc/profile.d/conda.sh && \
 101 | >>>     conda activate vitis-ai-tensorflow && \
 102 | >>>     cd $TVM_VAI_HOME/ &&\
 103 | >>>     python -m ck pull repo:ck-env && \
 104 | >>>     python -m ck install package:imagenet-2012-val-min --path=/opt/tvm-vai/CK-TOOLS
 105 |     
--------------------
ERROR: failed to solve: process "/bin/bash -c . $VAI_ROOT/conda/etc/profile.d/conda.sh &&     conda activate vitis-ai-tensorflow &&     cd $TVM_VAI_HOME/ &&    python -m ck pull repo:ck-env &&     python -m ck install package:imagenet-2012-val-min --path=/opt/tvm-vai/CK-TOOLS" did not complete successfully: exit code: 1
ERROR: docker build failed.

but the www.dropbox.com can be accessed by Chrome. and in terminal use wget command can download the ILSVRC2012_img_val_500.tar. how to solve this problem? anyone can help?

**Comment by gfursin** [comment]
By the way, I just realized that you are using a previous version of my automation framework (CK). We can still update it if there are users but I suggest you to try the next generation of CK that I have developed a few years ago (Collective Mind aka CM) and continue improving at MLCommons: https://github.com/mlcommons/ck/tree/master/cm  & https://github.com/mlcommons/ck/releases . We have fixes some of those URLs in the new framework but I need to double check.

**Comment by nathanw-mlc** [comment]
Hey folks, MLCommons has never officially used Dropbox to distribute those files, so I can't help there. I believe the latest version of CM/CK fetches the files from our official Cloudflare R2 buckets.

**Comment by gfursin** [comment]
Hey Nathan. Yes, that's correct. But I am curious if it makes sense to move such files from DropBox to MLCommons Cloudlare R2 buckets for the deprecated framework (if someone still uses it) or it's better/simpler to migrate to newer automation recipes from CM? I am neutral here.

**Comment by nathanw-mlc** [comment]
Ah, I see. I'd rather push people onto the newer versions than continue to support deprecated versions. We're running a pretty lean team here, so I don't think we have the bandwidth for that kind of support.

**Comment by Xuhd08** [comment]
OK, I try to update it to CM. Thanks!
